parent/pom.xml

```diff
@@ -552,6 +552,11 @@
                                 <packages>org.apache.ignite.spi.tracing.opencensus*</packages>
                             </group>
                         </groups>
+                        <header>
+                            <![CDATA[
```

Comment:
Why pom.xml was changed? It is not related to this issue.

Reply:
A mistake - removed this one.